src/ostree/ot-admin-builtin-upgrade.c

```diff
@@ -95,9 +95,6 @@ ot_admin_builtin_upgrade (int argc, char **argv, GCancellable *cancellable, GErr
                                                   "override-commit", NULL);
         }
 
-      /* Should we consider requiring --discard-hotfix here? */
-      origin_changed |= g_key_file_remove_key (origin, "origin", "unlocked", NULL);
```

Comment:
Yeah, but in the situation where a user has a hotfixed system, then upgrades to this ostree version (discarding the hotfix), the `unlocked` bit in the origin will be carried forward forever.  Yeah it's just one line in the file but still, it's easy for us to keep this code (with a `/* legacy */` comment).